modules/imaging_browser/php/NDB_Menu_Filter_imaging_browser.class.inc

```diff
@@ -149,7 +151,14 @@ class NDB_Menu_Filter_Imaging_Browser extends NDB_Menu_Filter
             'Links',
             'sessionID'
         );
-       
+
+        // Insert project column after DCCID, if useProject config is enabled
+        if ($config->getSetting('useProjects') === "true") {
```

Comment:
@driusan I used the same check as the one already used on **[Line 235](https://github.com/alisterdev/Loris/blob/4d6cb8575d6c8e14a9b0e291eb60c2ad4c7eb566/modules/imaging_browser/php/NDB_Menu_Filter_imaging_browser.class.inc#L235)**

Do you want me to change both to:
```php
if ($config->getSetting('useProjects')){...}
```
or 

```php
$useProject = $config->getSetting('useProjects');
if ($useProject == "true" || $useProject == "1"){...}
```

??